traffic_ops/app/db/migrations/20200227000000_add_ds_slice_block_size.sql

```diff
@@ -0,0 +1,22 @@
+/*
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE deliveryservice ADD COLUMN slice_plugin_block_size int DEFAULT NULL;
```

Comment:
IMO data model constraints should be implemented in the database (not that there's then anything wrong with checking at the API level) with a `CHECK` clause like:
```sql
ALTER TABLE deliveryservice
ADD COLUMN range_slice_block_size integer
CHECK (range_slice_block_size >= 262144 AND range_slice_block_size <= 33554432)
DEFAULT NULL;
```

Reply:
im indifferent - changed it to this ^5